config/samples/apps_v1alpha1_nemocustomizer.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: apps.nvidia.com/v1alpha1
+kind: NemoCustomizer
+metadata:
+  labels:
+    app.kubernetes.io/name: k8s-nim-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: nemocustomizer-sample
+spec:
+  wandbSecret:
+    name: wandb-secret
+    apiKeyKey: encryption_key
+  otel:
+    exporterOtlpEndpoint: http://<otel-collector-service>:4317
+  databaseConfig:
+    credentials:
+      user: ncsuser
+      secretName: ncs-pg-existing-secret
+      passwordKey: password
+    host: <db-service>
+    port: 5432
+    databaseName: ncsdb
+  expose:
+    service:
+      type: ClusterIP
+      ports:
+        - name: api
+          port: 8000
+          protocol: TCP
+        - name: internal
+          port: 9009
+          protocol: TCP
+  image:
+    repository: "nvidian/nemo-llm/customizer"
+    tag: "25.02-rc1"
+    pullPolicy: IfNotPresent
+    pullSecrets:
+      - ngc-image-pull-secret
+  replicas: 1
+  resources:
+    requests:
+      memory: "256Mi"
+      cpu: "500m"
+    limits:
+      memory: "512Mi"
+      cpu: "1"
+  livenessProbe:
+    enabled: true
+    probe:
+      failureThreshold: 5
+      httpGet:
+        path: /v1/health/live
+        port: api
+        scheme: HTTP
+      initialDelaySeconds: 15
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 10
+  readinessProbe:
+    enabled: true
+    probe:
+      failureThreshold: 5
+      httpGet:
+        path: /v1/health/ready
+        port: api
+        scheme: HTTP
+      initialDelaySeconds: 15
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 1
+  customizerConfig: |
```

Comment:
Just some thoughts -- if we let the customer to manage the configmap separately, it would be mostly the same complexity for them, but our code can be simpler

Reply:
initially thought of that approach, but we don't automatically reconcile of external configmap changes. Also, potentially the operator might need to auto add some custom values into the configmap as well.